contracts/mocks/ConstructorRebaseCallerContract.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.4.24;
+
+import "../UFragmentsPolicy.sol";
+
+
+contract ConstructorRebaseCallerContract {
+    UFragmentsPolicy private _policy;
```

Comment:
Don't need this property since it's used in the function it's passed into.

Reply:
Done 👍